src/main/java/seedu/address/model/Model.java

```diff
@@ -52,6 +52,21 @@
      */
     Path getAddressBookFilePath();
 
+    /**
+     * Returns the user prefs' ParticipantList file path.
+     */
+    Path getParticipantListFilePath();
```

Comment:
Is there a need to have this on both Logic and Model side? Asking cuz idk what this does lol

Reply:
I was intending to put in an extra layer of abstraction so that only Logic manages with file path, but it can be refactored out later on haha

Reply:
I was intending to put in an extra layer of abstraction so that only Logic manages with file path, but it can be refactored out later on haha

Reply:
I have the same concern as John :)